game.h

```diff
@@ -1,6 +1,6 @@
 #ifndef GAME_H
 #define GAME_H
 
-#define TICKS_PER_TURN 1000
+#include "game_cfg/game_cfg.h"
```

Comment:
So, I'm including this header in all the other files. That's probably bad despite using a guard? I don't know why it would be bad.

Reply:
No, that's fine. Having shared and widely used stuff in a single location is good.